lib/forge_sdk/conn/supervisor.ex

```diff
@@ -34,6 +34,10 @@ defmodule ForgeSdk.ConnSupervisor do
     DynamicSupervisor.which_children(__MODULE__)
   end
 
+  def get_names do
+    Enum.map(children(), fn {_, p, _, _} -> Process.info(p)[:registered_name] end)
```

Comment:
Process.info(p) will give lots of info including gc and other stuff, it's better to use Process.info(p, :registered_name) to only get info that needed.

Reply:
thanks!